platforms/ios/HelloCordova/Plugins/WizAssetsPlugin/WizAssetsPlugin.m

```diff
@@ -256,10 +256,10 @@ - (void)deleteFiles:(CDVInvokedUrlCommand *)command {
 
 - (void)backgroundDelete:(CDVInvokedUrlCommand *)command {
     @autoreleasepool {
-        NSMutableArray *uris = [[NSMutableArray alloc] initWithArray:command.arguments copyItems:YES];
+        NSArray *uris = command.arguments;
```

Comment:
Indeed, this was legacy code of deleting files when it was not yet done in the background. There should not be any weird multi threaded cases here.